cocos/core/geometry/deprecated-3.0.0.ts

```diff
@@ -151,8 +151,12 @@ function deprecatedClassMessage (oldClassName: string, newClassName) {
 }
 
 /**
+ * @en
  * Alias of [[Line]]
- * @deprecated Since v3.0
+ * @zh
+ * [[Line]] 别名类
+ *
+ * @deprecated Since v3.0, please use Line instead
```

Comment:
Add `@internal` for all classes in this file

Reply:
@internal will cause compiling failed